docs/cli-client/bank/README.md

```diff
@@ -1 +1,47 @@
-# iriscli 
\ No newline at end of file
+# iriscli bank
+
+## Description
+
+Bank allows you to management your local account 
```

Comment:
`management` is a noun, you could use `manage` instead, but `bank` is not use to manage your local account, maybe `manage assets` is better.